### PR TITLE
Droth 1556 changing zoom level and labels on oth new version

### PR DESCRIPTION
**Pull request by oskarmalmstrom**

### CODE REVIEW

File: digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/LinearAssetPartitioner.scala [hunk 1]
```diff
@@ -19,4 +19,24 @@ object LinearAssetPartitioner extends GraphPartitioner {
...
+    }
+
+    val (linksToPartition, linksToPass) = linkGroups.partition { case ((_, _, _), _) => true }
+
+    val clusters = linksToPartition.values.map(p => {
+      clusterLinks(p)
+    }).toSeq.flatten
+
+
+    //val clusters = for (linkGroup <- linksToPartition.values.toSeq;
```
Comment (moraispedro) [inline digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/LinearAssetPartitioner.scala]: Comment can be deleted if not needed.

File: digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/LinearAssetPartitioner.scala [hunk 2]
```diff
@@ -19,4 +19,24 @@ object LinearAssetPartitioner extends GraphPartitioner {
...
+    //                    cluster <- clusterLinks(linkGroup)) yield cluster
```
Comment (moraispedro) [inline digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/LinearAssetPartitioner.scala]: Comment can be deleted if not needed.

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/linearasset/OracleLinearAssetDao.scala [hunk 3]
```diff
@@ -379,6 +415,29 @@ class OracleLinearAssetDao(val vvhClient: VVHClient, val roadLinkService: RoadLi
     }
   }
 
+  def fetchLinearAssets(assetTypeId: Int, valuePropertyId: String, linkSource: Option[LinkGeomSource] = None): Seq[PieceWiseLinearAsset] = {
+    val linkGeomCondition = linkSource match {
+      case Some(LinkGeomSource.NormalLinkInterface) => " and pos.link_source = 1"
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/linearasset/OracleLinearAssetDao.scala]: case Some(LinkGeomSource.NormalLinkInterface) => " and pos.link_source = ${LinkGeomSource.NormalLinkInterface.value}"

**Comment by oskarmalmstrom** [comment]
ok

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/LinearAssetService.scala [hunk 4]
```diff
@@ -550,11 +558,11 @@ trait LinearAssetOperations {
             case (Some(createdBy), Some(createdDateTime)) =>
               dao.createLinearAsset(linearAsset.typeId, linearAsset.linkId, linearAsset.expired, linearAsset.sideCode,
                 Measures(linearAsset.startMeasure, linearAsset.endMeasure), LinearAssetTypes.VvhGenerated, linearAsset.vvhTimeStamp,
-                getLinkSource(roadLinks.find(_.linkId == linearAsset.linkId)), fromUpdate = true, Some(createdBy), Some(createdDateTime), linearAsset.verifiedBy, linearAsset.verifiedDate)
+                getLinkSource(roadLinks.find(_.linkId == linearAsset.linkId)), fromUpdate = true, Some(createdBy), Some(createdDateTime), linearAsset.verifiedBy, linearAsset.verifiedDate, geometry = getGeometry(roadLinks.find(_.linkId == linearAsset.linkId)))
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/LinearAssetService.scala]: Before line 556 add val roadlink =  roadLinks.find(_.linkId == linearAsset.linkId)

and replace  roadLinks.find(_.linkId == linearAsset.linkId) -> roadlink

... getLinkSource(roadLink), fromUpdate = true, Some(createdBy), Some(createdDateTime), linearAsset.verifiedBy, linearAsset.verifiedDate, geometry  = getGeometry(roadLink))

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/LinearAssetService.scala [hunk 5]
```diff
@@ -550,11 +558,11 @@ trait LinearAssetOperations {
...
             case _ =>
               dao.createLinearAsset(linearAsset.typeId, linearAsset.linkId, linearAsset.expired, linearAsset.sideCode,
                 Measures(linearAsset.startMeasure, linearAsset.endMeasure), LinearAssetTypes.VvhGenerated, linearAsset.vvhTimeStamp,
-                getLinkSource(roadLinks.find(_.linkId == linearAsset.linkId)))
+                getLinkSource(roadLinks.find(_.linkId == linearAsset.linkId)), geometry = getGeometry(roadLinks.find(_.linkId == linearAsset.linkId)))
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/LinearAssetService.scala]: lets replace roadLinks.find(_.linkId == linearAsset.linkId) -> roadlink

getLinkSource(roadLink), geometry = getGeometry(roadLink))

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/RoadWidthService.scala [hunk 6]
```diff
@@ -148,11 +148,11 @@ class RoadWidthService(roadLinkServiceImpl: RoadLinkService, eventBusImpl: Digir
           case (Some(createdBy), Some(createdDateTime)) =>
             dao.createLinearAsset(linearAsset.typeId, linearAsset.linkId, linearAsset.expired, linearAsset.sideCode,
               Measures(linearAsset.startMeasure, linearAsset.endMeasure), linearAsset.modifiedBy.getOrElse(LinearAssetTypes.VvhGenerated), linearAsset.vvhTimeStamp,
-              getLinkSource(roadLinks.find(_.linkId == linearAsset.linkId)), fromUpdate = true, Some(createdBy), Some(createdDateTime), linearAsset.verifiedBy, linearAsset.verifiedDate, Some(MmlNls.value))
+              getLinkSource(roadLinks.find(_.linkId == linearAsset.linkId)), fromUpdate = true, Some(createdBy), Some(createdDateTime), linearAsset.verifiedBy, linearAsset.verifiedDate, Some(MmlNls.value), geometry = getGeometry(roadLinks.find(_.linkId == linearAsset.linkId)))
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/RoadWidthService.scala]: lets replace roadLinks.find(_.linkId == linearAsset.linkId) -> roadlink

getLinkSource(roadLink), ..., geometry = getGeometry(roadLink))

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/RoadWidthService.scala [hunk 7]
```diff
@@ -148,11 +148,11 @@ class RoadWidthService(roadLinkServiceImpl: RoadLinkService, eventBusImpl: Digir
...
           case _ =>
             dao.createLinearAsset(linearAsset.typeId, linearAsset.linkId, linearAsset.expired, linearAsset.sideCode,
               Measures(linearAsset.startMeasure, linearAsset.endMeasure), linearAsset.createdBy.getOrElse(LinearAssetTypes.VvhGenerated), linearAsset.vvhTimeStamp,
-              getLinkSource(roadLinks.find(_.linkId == linearAsset.linkId)), verifiedBy = linearAsset.verifiedBy, informationSource = Some(MmlNls.value))
+              getLinkSource(roadLinks.find(_.linkId == linearAsset.linkId)), verifiedBy = linearAsset.verifiedBy, informationSource = Some(MmlNls.value), geometry = getGeometry(roadLinks.find(_.linkId == linearAsset.linkId)))
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/linearasset/RoadWidthService.scala]: lets replace roadLinks.find(_.linkId == linearAsset.linkId) -> roadlink

getLinkSource(roadLink), ..., geometry = getGeometry(roadLink))

File: UI/src/view/providers/layer.js [hunk 8]
```diff
@@ -1,5 +1,5 @@
 (function(root) {
-  root.Layer = function(layerName, roadLayer) {
+  root.Layer = function(layerName, roadLayer, highZoomLevel) {
```
Comment (marquesfm) [inline UI/src/view/providers/layer.js]: "highZoomLevel" never used in this place.

**Pull request merged**